build.yaml

```diff
@@ -2007,3 +2009,73 @@ steps:
      - genetics_public_image
    scopes:
     - dev
+ - kind: buildImage
+   name: address_image
+   dockerFile: address/Dockerfile
+   contextPath: .
+   publishAs: address
+   dependsOn:
+     - service_base_image
+ - kind: runImage
+   name: check_address
+   image:
+     valueFrom: address_image.image
+   script: |
+     set -ex
+     python3 -m flake8 address
+     python3 -m pylint --rcfile pylintrc address
+   dependsOn:
+     - address_image
+ - kind: deploy
+   name: deploy_address_sa
+   namespace:
+     valueFrom: default_ns.name
+   config: address/service-account.yaml
+   dependsOn:
+    - default_ns
+ - kind: deploy
+   name: deploy_address
+   namespace:
+     valueFrom: default_ns.name
+   config: address/deployment.yaml
+   wait:
+    - kind: Service
+      name: address
+      for: alive
+   dependsOn:
+    - default_ns
+    - deploy_router
+    - address_image
+    - create_certs
+ - kind: runImage
+   name: test_address
+   image:
+     valueFrom: service_base_image.image
+   script: |
+     python3 -m pytest --log-date-format="%Y-%m-%dT%H:%M:%S" --log-format="%(asctime)s %(levelname)s %(name)s %(filename)s:%(lineno)d:%(funcName)s %(message)s" --log-cli-level=INFO -s -vv --instafail /io/test/
```

Comment:
Do you want this to be instafail?

Reply:
Yes, instafail shows errors/failures as soon as the happen instead of at the end of all the tests. I copied this from batch.

https://github.com/pytest-dev/pytest-instafail